src/data/providers.json

```diff
@@ -0,0 +1,307 @@
+{
+  "object": "list",
+  "version": "1.0.0",
+  "data": [
+    {
+      "id": "ai21",
```

Comment:
would recommend using the enums defined in `/src/globals` for all the providerIds in this file and in `models.json`